src/dvc_data/hashfile/meta.py

```diff
@@ -79,12 +80,32 @@ def from_dict(cls, d: Dict[str, Any]) -> "Meta":
         return cls(**kwargs)
 
     def to_dict(self) -> Dict[str, Any]:
-        def _filter(field: Attribute, value: Any) -> bool:
-            if field.name in (self.PARAM_INODE, self.PARAM_MTIME):
-                return False
-            return _filter_default_or_none(field, value)
+        ret: dict = OrderedDict()
```

Comment:
You can probably drop ~40% more by using just a dict.
```suggestion
        ret = {}
```

Reply:
Also, you can get rid of `_filter_default_or_none` if we go this route.